samples/bluetooth/direct_test_mode/src/fem/nrf21540.c

```diff
@@ -186,21 +186,26 @@ static uint32_t radio_tx_ramp_up_delay_get(bool fast, nrf_radio_mode_t mode)
 		ramp_up = fast ? RADIO_RAMP_UP_TX_2M_FAST_US :
 				 RADIO_RAMP_UP_TX_2M_US;
 		break;
-
+#if defined(RADIO_MODE_MODE_Ble_LR125Kbit)
```

Comment:
IMO it would be better to use Kconfig symbels instead of nrf header defines.
I suggest using `CONFIG_HAS_HW_NRF_RADIO_BLE_CODED` instead.

Reply:
Good idea. I changed it for dtm and radio test as well in all places